index.js

```diff
@@ -94,16 +94,20 @@ exports.kinesisHandler = function (records, context, callback) {
     request(options, function (error, response, body) {
       logger.info({'message': 'Posting........'})
       logger.info({'message': 'Response: ' + response.statusCode + "  Records info:" + records})
-      if (response.statusCode !== 200) {
+      
+      if ([500, 401].include?(response.statusCode)) {
         if (response.statusCode === 401) {
           // Clear access token so new one will be requested on retried request
           CACHE['accessToken'] = null
         }
         callback(new Error())
         logger.error({'message': 'POST Error! ', 'response': response})
         return
+      } else if ([400, 404].include?(response.statusCode)) {
+        logger.error({'message': 'Post api input validations failed!', 'response': response})
```

Comment:
This PR is to add more logs. If logs are not required we can close this PR.

Reply:
The `error` messages are useful. But let's delete the `info` messages.